pyseir/deployment/demographic_mapper.py

```diff
@@ -0,0 +1,592 @@
+        as secondary key.
+
+
+    Parameters
+    ----------
+    fips: str
+        State of county FIPS code
+    mle_model: SEIRModelAge
+        Model with age structure and MLE model parameters
+    fit_results: dict
+        MLE epi parameters and associated errors. The parameter used is
+        t0_date.
+    measures: str or list(str)
+        Names of covid measures, should be interpretable by CovidMeasure.
+    measure_units: str or list(str)
+        Units of covid measures, should be interpretable by CovidMeasureUnit.
+    target_age_distribution: callable
+        PDF of age of target population.
+    risk_modifier_by_age: callable
+        Returns risk rations by age group that modifies the risk of
+        hospitalization or mortality.
+    """
+
+    def __init__(self,
+                 fips,
+                 mle_model,
+                 fit_results,
+                 measures,
+                 measure_units,
+                 target_age_distribution=None,
+                 risk_modifier_by_age=None):
+
+        self.fips = fips
+        self.predictions = {k: v for k, v in mle_model.results.items() if k != 'by_age'}
+        self.predictions_by_age = mle_model.results['by_age']
+        self.parameters = {k: v for k, v in mle_model.__dict__.items() if k not in ('by_age', 'results')}
+        self.fit_results = fit_results
+        self.measures = [measures] if not isinstance(measures,list) else measures
+        self.measures = [CovidMeasure(m) for m in self.measures]
+
+        self.measure_units = [measure_units] if not isinstance(measure_units,list) else measure_units
+        self.measure_units = [CovidMeasureUnit(u) for u in self.measure_units]
+
+        if target_age_distribution is None:
+            target_age_distribution = lambda x: np.array([1] * len(self.parameters['age_groups']))
+        self.target_age_distribution = target_age_distribution
+        self.risk_modifier_by_age = risk_modifier_by_age
+
+        # get parameters required to calculate covid measures
+        self.hospitalization_rates, self.mortality_rates = self._generate_hospitalization_mortality_rates()
+        self.prevalence = self._age_specific_prevalence()
+
+        self.mapped_predictions = None
+
+
+    def _age_specific_prevalence(self):
+        """
+        Calculate age specific prevalence.
+
+        Returns
+        -------
+        prevalence: np.array
+            Trajectory of covid infection prevalence inferred by the MLE
+            model.
+        """
+        prevalence = (self.predictions_by_age['I']
+                    + self.predictions_by_age['A'])/ self.parameters['N'][:, np.newaxis]
+        return prevalence
+
+    def _reconstruct_mortality_rates_trajectory(self):
+        """
+        Reconstruct trajectory of inferred per-capita mortality rates through
+        time from MLE model parameters and compartments.
+
+        Returns
+        -------
+        mortality_rate_general: np.array
```

Comment:
Since this has led to confusion, why don't we call this `mortality_inflow_rate`

Reply:
Good idea!